src/main/java/com/redhat/pantheon/sling/PantheonRepositoryInitializer.java

```diff
@@ -50,16 +50,8 @@ private void initializeRepositoryACLs(SlingRepository slingRepository) throws Re
                 log.info("Pantheon service account already exists");
             }
             assignPermissionToPrincipal(s, "pantheon", "/content", "*", Privilege.JCR_ALL);
-
-            // Create and grant permissions to the pantheon-users group
-            try {
-                s.getUserManager().createGroup("pantheon-users");
-                s.save();
-                log.info("Created pantheon-users group");
-            } catch (AuthorizableExistsException aeex) {
-                log.info("pantheon-users group already exists");
-            }
-            assignPermissionToPrincipal(s,"pantheon-users","/content/repositories", null, Privilege.JCR_ALL);
+            assignPermissionToPrincipal(s,"pantheon-users","/content/repositories", null, Privilege.JCR_WRITE, Privilege.JCR_NODE_TYPE_MANAGEMENT);
+            assignPermissionToPrincipal(s,"pantheon-users","/content/modules", null, Privilege.JCR_WRITE, Privilege.JCR_NODE_TYPE_MANAGEMENT);
```

Comment:
I think we need to decide where we want to keep our content. Either `repositories` or `modules` (personally think it should be the former).

Reply:
Yes, we need to decide that. My concern for this PR was getting the codebase working again with as few changes as possible, not to finalize our folder structure.